cocos/platform/ios/CCDirectorCaller-ios.mm

```diff
@@ -137,7 +138,10 @@ -(void) doCaller: (id) sender
         
         [EAGLContext setCurrentContext: cocos2dxContext];
 
-        CFTimeInterval dt = ((CADisplayLink*)displayLink).timestamp - lastDisplayTime;
+        CFTimeInterval dt = 0;
+        if (lastDisplayTime != 0) {
+            dt = ((CADisplayLink*)displayLink).timestamp - lastDisplayTime;
+        }
```

Comment:
Oh, can not be `1/60`, but may be initialize with a timestamp. What i think it is better to avoid the `if` codes in every frame.